### DISCUSSION TITLE
AzuraCast PWA Player is OUT

### DISCUSSION THREAD

**Discussion by oussamatn** [AzuraCast/AzuraCast#4566] [Show and tell]
Hello guys,

It's finally here ! The web player for Azuracast.  
Plug&play fully integrated with Azuracast API .

> **Project repository :** 
> [https://github.com/oussamatn/radio-player](https://github.com/oussamatn/radio-player)

I have been working on this a years ago, then I forgot about it for a while. ( #1937 #4132)
Well, Today is a new day ! The player is available for beta tester. 
Don't hesitate to make suggestion, pull requests, and bugs report.

> Here's a short preview : 
> [https://player.jelliti.me](https://bit.ly/3uCCNaf)

Screenshots : 

<img src="https://user-images.githubusercontent.com/2962369/117323879-92b7af80-ae8f-11eb-9411-90fe1bc0459a.gif" width="300"><img src="https://user-images.githubusercontent.com/2962369/117322713-8a12a980-ae8e-11eb-8fe1-34106f25fe25.png" width="300"><img src="https://user-images.githubusercontent.com/2962369/117321797-bc6fd700-ae8d-11eb-997c-4f46c67fbbfc.png" width="300">

**Comment by ultragain** [comment]
nice done!

**Comment by pixel-paul** [comment]
Is this based upon the Soma FM Vue app?
